packages/firebase_remote_config/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -13,43 +11,34 @@ part of firebase_remote_config;
 // ignore: prefer_mixin
 class RemoteConfig extends FirebasePluginPlatform with ChangeNotifier {
   RemoteConfig._({this.app})
-      : super(app.name, 'plugins.flutter.io/firebase_remote_config');
+      : super(app!.name, 'plugins.flutter.io/firebase_remote_config');
 
   // Cached instances of [FirebaseRemoteConfig].
   static final Map<String, RemoteConfig> _firebaseRemoteConfigInstances = {};
 
-  // Cached and lazily loaded instance of [FirebaseRemoteConfigPlatform]
-  // to avoid creating a [MethodChannelFirebaseRemoteConfig] when not needed
-  // or creating an instance with the default app before a user specifies an
-  // app.
-  FirebaseRemoteConfigPlatform _delegatePackingProperty;
-
   /// Returns the underlying delegate implementation.
   ///
   /// If called and no [_delegatePackingProperty] exists, it will first be
   /// created and assigned before returning the delegate.
   FirebaseRemoteConfigPlatform get _delegate {
-    return _delegatePackingProperty ??=
-        FirebaseRemoteConfigPlatform.instanceFor(
+    return FirebaseRemoteConfigPlatform.instanceFor(
```

Comment:
Why was the local cache removed? This changes the behavior.

Did you forget to change it from a getter to a `late final` variable?

Reply:
had removed in the last cleaning, adjusted.